npm/wizer/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@bytecode-alliance/wizer",
+  "version": "1.5.56",
+  "description": "The WebAssembly Pre-Initializer",
+  "type": "module",
+  "scripts": {
+    "update": "node ./update.js",
+    "postinstall": "node ./post-install.js"
```

Comment:
Nit and feel free to ignore: Post-install hooks are considered somewhat of a bad pattern these days and more users are disabling them for security. Wondering if it might just make sense to do a runtime stat over node_modules to see what's there?